sonic-ycabled/tests/mock_platform.py

```diff
@@ -0,0 +1,262 @@
+class MockDevice:
```

Comment:
This file is not used by y-cable tests. seems this file is copied from 'sonic-xcvrd/tests/mock_platform.py' .

Reply:
removed